files/saslauthd-postfix

```diff
@@ -0,0 +1,44 @@
+# {{ ansible_managed }}
```

Comment:
This file is not a template, it's just copied to the host.

Reply:
I removed the line.

Reply:
Why remove?
The comment is still useful.
I'd suggest just hardcoding the message. 
Alternatively: Template that file jsut for this (we did this before)

Reply:
Done